src/main/python/janusgraph_python/core/attribute/TextPredicate/Text.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from gremlin_python.process.traversal import P
+
+
+class Text(object):
+    """
+    Class implementing Gremlin Python's Predicate class, and extending it for JanusGraph based Text Predicates.
```

Comment:
_Class implementing Gremlin Python's Predicate class_ <- Is this class really implementing Gremlin-Python's `P` class? I think that it only provides JanusGraph's Text predicates.

Reply:
I've changed that to `This class implements all the JanusGraph based Text Predicates.` and similarly for Geo too.